UI/src/view/csvReportsWorkList.js

```diff
@@ -0,0 +1,489 @@
+(function (root) {
+  root.csvReportsWorkList = function () {
```

Comment:
Normally we uppercase the first letter, maybe it could negligible...

Reply:
My mistake.

Reply:
Fixed